src/engine/game/game.cpp

```
@@ -32,125 +34,125 @@ void Game::Initialize()
+  //   return;
+  // }
+  // this->renderManager->Render(*this->renderer);
+  //
+  // this->userInterfaceManager->DebugRender();
 }
 
 Engine::Scenes::SceneManager& Game::GetSceneManager()
 {
-  return this->sceneManager;
+  // return this->sceneManager;
 }
```

Cppcheck (2.20.0)

[error] Found an exit path from function with non-void return type that has missing return statement

(missingReturn)